src/test/resources/xmpp_client_configs.conf

```diff
@@ -0,0 +1,30 @@
+multiple-muc-config {
+  videobridge {
+    apis {
+      xmpp-client {
+        configs {
+          shard-1 {
+            "DISABLE_CERTIFICATE_VERIFICATION" = true
+            "DOMAIN" = auth.brian.jitsi.net
+            "PASSWORD" = "password\\+"
+            "HOSTNAME" = brian.jitsi.net
+            "MUC" = "JvbBrewery@internal.auth.brian.jitsi.net"
+            "MUC_JIDS" = "JvbBrewery@internal.auth.brian.jitsi.net"
+            "MUC_NICKNAME"=brian_local
+            "USERNAME"=username
+          }
+          shard-2 {
+            "DISABLE_CERTIFICATE_VERIFICATION" = true
+            "DOMAIN" = auth.brian2.jitsi.net
+            "PASSWORD" = "password\\+"
+            "HOSTNAME" = brian2.jitsi.net
+            "MUC" = "JvbBrewery@internal.auth.brian2.jitsi.net"
+            "MUC_JIDS" = "JvbBrewery@internal.auth.brian2.jitsi.net"
+            "MUC_NICKNAME"=brian_local2
```

Comment:
This doesn't have to be different from the one in shard-1. I'd use the same value to avoid implying that has to be different. I'd use something like `jvb-1` in both places to suggest it's an identifier of the bridge

Reply:
For the purpose of a test, I think it's better to use distinct values...otherwise it's hard to ensure that both chunks were actually parsed.  I get that it doesn't necessarily mirror the real-life application, but this is more about testing the borders of the logic.